src/frontend/playwright.config.ts

```diff
@@ -1,14 +1,18 @@
 import { defineConfig, devices } from "@playwright/test";
+import * as dotenv from "dotenv";
+import path from "path";
 
 /**
  * Read environment variables from file.
  * https://github.com/motdotla/dotenv
  */
-// require('dotenv').config();
-
+dotenv.config();
 /**
  * See https://playwright.dev/docs/test-configuration.
  */
+
+dotenv.config({ path: path.resolve(__dirname, "../../.env") });
```

Comment:
Will this work in CI?

Reply:
Actually, I had to tweak that because the Playwright frontend couldn't even see the .env. So, I had to install dotenv and set up Playwright to chat with the .env so I could run tests using API keys. Not sure if it'll work on CI, but it should since it's now actually picking up the .env correctly.
@ogabrielluiz